.github/workflows/rebase.yml

```diff
@@ -0,0 +1,26 @@
+name: Automatic Rebase
+on:
+  issue_comment:
+    types: [created]
+jobs:  
+  rebase:
+    name: Rebase
+    runs-on: ubuntu-latest
+    if: >-
+      github.event.issue.pull_request != '' && 
```

Comment:
I think we'll want to restrict this here to collaborators (people who have push access) https://github.com/cirrus-actions/rebase#restricting-who-can-call-the-action

I believe the permission we want is "COLLABORATOR" not "MEMBER"